runtimes/eden/src/pallets_governance.rs

```diff
@@ -32,7 +32,7 @@ impl pallet_reserve::Config<pallet_reserve::Instance1> for Runtime {
 	type ExternalOrigin = MoreThanHalfOfTechComm;
 	type Call = Call;
 	type PalletId = CompanyReservePalletId;
-	type WeightInfo = pallet_reserve::weights::SubstrateWeight<Runtime>;
+	type WeightInfo = crate::weights::pallet_reserve::WeightInfo<Runtime>;
```

Comment:
pallet_reserve is internal to us and it could look into pallet_reserve's own folder.